pkg/pmaperrors/pmaperrors.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 PAMP authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pmaperrors defines the error wrapper for user facing errors.
+package pmaperrors
+
+import (
+	"errors"
+	"fmt"
+)
+
+type processError struct {
+	err error
+}
+
+// New() takes an string, and return a precessError
```

Comment:
nit: you don't need the parenthesis. Throughout
```suggestion
// New takes an string, and return a precessError
```